src/style.css

```diff
@@ -0,0 +1,35 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+}
```

Comment:
I would suggest moving `list-style: none` to the top and reseting it for all `<ul>`